spec/models/user_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  it "has a valid user" do
+    expect(build(:user)).to be_valid
+  end
+
+  describe "associations" do
+
+    it "have many comments" do
+      is_expected.to have_many(:comments).dependent :destroy
+    end
+
+    it "have many suggests" do
+      is_expected.to have_many(:suggests).dependent :destroy
+    end
+
+    it "have many favorites" do
+      is_expected.to have_many(:favorites).dependent :destroy
+    end
+
+    it "have many reviews" do
+      is_expected.to have_many(:reviews).dependent :destroy
+    end
+
+    context "presence" do
+      it {should validate_presence_of :name}
+    end
+  end
+
+  describe "validate" do
+    let(:user){FactoryBot.create :user}
+
+    describe "name" do
+      it {is_expected.to validate_presence_of(:name)}
+      it "can't be blank" do
+        user.name = nil
+        user.valid?
+        expect(user.errors.messages[:name].first).to eq(
+          I18n.t "activerecord.errors.models.user.attributes.name.blank")
```

Comment:
sửa thành `it {is_expected.to validate_presence_of(:name.with_message(I18n.t("activerecord.errors.models.user.attributes.name.blank"))}` được ko

Reply:
dạ nếu sửa thành **it {is_expected.to validate_presence_of(:name).with_message(
I18n.t "activerecord.errors.models.user.attributes.name.blank")}**  thì vẫn được ạ